packages/react/src/themes/teams/index.tsx

```diff
@@ -36,6 +36,7 @@ const themeIcons: ThemeIcons = Object.keys(svgIconsAndStyles as {
 
 const icons: ThemeIcons = {
   ...themeIcons,
+  'stardust-checkmark': themeIcons['stardust-checkmark'],
```

Comment:
Can't we just use the `accept` icon from teams theme?

Reply:
Overlooked it, definitely it's better to use `accept` 👍